src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import { HStack, Flex } from "@chakra-ui/react"
+import { PropsWithChildren } from "react"
+
+export const Footer = ({
+  children,
+}: PropsWithChildren<Record<never, any>>): JSX.Element => (
+  <Flex
+    h="5rem"
+    w="100%"
+    pr="2rem"
+    position="fixed"
```

Comment:
i don't see a problem with this but just wanted to point out that fixed removes it from the document flow (see [here](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/)). i think it might be worth asking designers if the page is guaranteed to provide space for footer (when it's positioned) or otherwise, using sticky to ensure that this element is accounted for/using grid layout to reserve space